python/foxglove-sdk/src/websocket.rs

```diff
@@ -379,12 +379,15 @@ pub fn start_server(
     services: Option<Vec<PyService>>,
     asset_handler: Option<Py<PyAny>>,
     context: Option<PyRef<PyContext>>,
+    session_id: Option<String>,
 ) -> PyResult<PyWebSocketServer> {
-    let session_id = time::SystemTime::now()
-        .duration_since(time::UNIX_EPOCH)
-        .expect("Failed to create session ID; invalid system time")
-        .as_millis()
-        .to_string();
+    let session_id = session_id.unwrap_or_else(|| {
+        time::SystemTime::now()
+            .duration_since(time::UNIX_EPOCH)
+            .expect("Failed to create session ID; invalid system time")
+            .as_millis()
```

Comment:
Curious why not as nanos? Could replace this with nanoseconds_since_epoch().to_string()?